fsspec/implementations/dirfs.py

```diff
@@ -329,13 +329,15 @@ def makedirs(self, path, *args, **kwargs):
     def rmdir(self, path):
         return self.fs.rmdir(self._join(path))
 
-    def mv_file(self, path1, path2, **kwargs):
-        return self.fs.mv_file(
+    def mv(self, path1, path2, **kwargs):
+        return self.fs.mv(
             self._join(path1),
             self._join(path2),
             **kwargs,
         )
 
+    mv_file = mv
```

Comment:
Does anything call mv_file? In most cases, the `_file` means it operates on a single file at a time, whereas the one without does possible lists and glob expansion. Perhaps we need both?